core/spec.md

```diff
@@ -1058,24 +1055,15 @@ When updating an entity that can contain Registry collections, the request
 MAY contain the 3 collection attributes. The `COLLECTIONSurl` and
 `COLLECTIONScount` attributes MUST be silently ignored by the server.
 
-A request that includes `COLLECTIONS` attributes MUST generate an error
-if the `?nested` query parameter is not present. This acts as a check to
-ensure the client is not accidentally updating the children entities.
-
-If the `?nested` query parameter and the `COLLECTIONS` attribute are both
-present, the server MUST process each entity in the collection map as a
-request to create or update that entity according to the semantics of the HTTP
-method used. An entry in the map that isn't a valid entity (e.g. is `null`)
-MUST generate an error.
-
-The `?nested` semantics MUST be applied to all nested `COLLECTIONS` attributes
-within the request regardless of the level in the hierarchy in which they
-appear.
+If `COLLECTIONS` attribute is present, the server MUST process each entity in
```

Comment:
I'd have either 

> If `COLLECTIONS` attributes are present

or

> If the `COLLECTIONS` attribute is present

The singular with no article feels a bit odd.

Reply:
done